tests/conftest.py

```diff
@@ -244,10 +244,14 @@ def make_package(tmp_path):
     Make a package from a given name, version and list of required packages.
     """
 
-    def _make_package(name, version="0.1", install_requires=None):
+    def _make_package(name, version="0.1", install_requires=None, extras_require=None):
+
         if install_requires is None:
             install_requires = []
 
+        if extras_require is None:
+            extras_require = dict()
```

Comment:
Nit: it's best to use a literal
```suggestion
            extras_require = {}
```